test/jsToAqua.test.ts

```diff
@@ -80,6 +60,12 @@ describe("Conversion from js to aqua", () => {
     }
   );
 
+  test("Expect objects to match snapshot", () => {
+    obj.forEach((o) => {
+      expect(jsToAquaImpl(o, WRAPPER_FUNCTION_NAME, "")).toMatchSnapshot();
+    });
+  });
```

Comment:
No idea how to review these tests. Need you help on that.

Reply:
No snapshots are better, you can check them out. What this is does is it generates a bunch of snapshots so you can look at the input and output and decide if they are correct